services/harmony/test/labels/label_crud.ts

```diff
@@ -1,12 +1,43 @@
 import { expect } from 'chai';
 import { profanity } from '@2toad/profanity';
-import { hookTransaction } from '../helpers/db';
+import { hookTransaction, truncateAll } from '../helpers/db';
 import { buildJob, getFirstJob } from '../helpers/jobs';
 import { addJobsLabels, deleteJobsLabels } from '../helpers/labels';
 import hookServersStartStop from '../helpers/servers';
 import db from '../../app/util/db';
 import env from '../../app/util/env';
 import { stub } from 'sinon';
+import { getLabelsForUser } from '../../app/models/label';
+
+describe('Get Labels', function () {
+  const joeJob = buildJob({ username: 'joe' });
+  const jillJob = buildJob({ username: 'jill' });
+  hookServersStartStop({ skipEarthdataLogin: false });
+  before(async function () {
+    await truncateAll();
+    const trx = await db.transaction();
+    await joeJob.save(trx);
+    await jillJob.save(trx);
+    await trx.commit();
+  });
+
+  describe('When getting labels using the admin route', function () {
```

Comment:
Seems like one more test should be added to sanity check the order is based on the most recent time a label has been used by any user.

Reply:
Done